File: libgdx_new_super_jumper/android/src/com/nopalsoft/superjumper/AndroidLauncher.java
package com.nopalsoft.superjumper;

import android.os.Bundle;

import com.badlogic.gdx.backends.android.AndroidApplication;
import com.badlogic.gdx.backends.android.AndroidApplicationConfiguration;
import com.nopalsoft.superjumper.handlers.GameServicesHandler;

public class AndroidLauncher extends AndroidApplication implements  GameServicesHandler {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        AndroidApplicationConfiguration config = new AndroidApplicationConfiguration();
        initialize(new MainSuperJumper( this), config);
    }






    @Override
    public void submitScore(long score) {

    }

    @Override
    public void getLeaderboard() {

    }


    @Override
    public boolean isSignedIn() {
        return false;
    }

    @Override
    public void signIn() {

    }

}

File: libgdx_galaxy_invaders/android/src/com/nopalsoft/invaders/AndroidLauncher.kt
package com.nopalsoft.invaders

import android.os.Bundle
import com.badlogic.gdx.backends.android.AndroidApplication
import com.badlogic.gdx.backends.android.AndroidApplicationConfiguration

class AndroidLauncher : AndroidApplication() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val config = AndroidApplicationConfiguration()
        initialize(MainInvaders(), config)
    }

    override fun onBackPressed() {
        // todo
    }
}
